charters/DataOperations/charter.md

```diff
@@ -0,0 +1,92 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team is responsible for providing the scientific community with effective tools and resources which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution.
+
+## Definitions
+
+Data contribution: a set of raw data derived from the same experimental process, and the metadata describing that process
+Data release: the point at which metadata and data from a single data contribution become publicly available
+Data distribution: the packaging of metadata and data from multiple data contributions in meaningful ways and any announcements or documentation supporting it
+
```

Comment:
When looking at this section on the github preview the words being defined could do with some emphasis, generally italics is used in other places and there aren't enough spaces at the end of lines to provide new lines so they are all running into one

Reply:
Should be more clear in 27f2299